utils/wasm-builder/src/wasm_project.rs

```diff
@@ -179,9 +180,16 @@ impl WasmProject {
             .join(format!("{}.meta.wasm", &file_base_name));
         Self::generate_meta(&from_path, &to_meta_path)?;
 
-        let wasm_binary_path = self.out_dir.join("wasm_binary_path.txt");
-        fs::write(&wasm_binary_path, format!("{}", to_opt_path.display()))
-            .context("unable to write `wasm_binary_path.txt`")?;
+        let wasm_binary_path = self.original_dir.join(".binpath");
```

Comment:
Am I right that since now every smart contract project should individually git-ignore the `.binpath` dir?

Reply:
It's not necessary, because .binpath generates every time and doesn't contain any personal info.
But in general, you're right. I don't see there any other workarounds.
I hope in future we gonna provide some cargo tooling which will generate base smart contracts by `cargo gear new` etc. (in #22)